components/sharing/SocialShare.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <b-link
+      v-for="(network, index) in networks"
+      :key="index"
+      :title="$t('actions.shareOn', { social: network.name })"
+      :href="network.url"
+      :class="`social-share mr-1 ${network.identifier}`"
+      :data-qa="`share ${network.identifier} button`"
+    >
+      <span :class="`icon-${network.identifier}`" />
+      <span
+        v-if="withText"
+        class="text"
+      >{{ network.name }}
+      </span>
+    </b-link>
+  </div>
+</template>
+
+<script>
+  import { mapGetters } from 'vuex';
+
+  export default {
+    name: 'SocialShare',
+
+    props: {
+      mediaUrl: {
+        type: String,
+        default: ''
+      },
+      withText: {
+        type: Boolean,
+        default: false
+      }
+    },
+
+    data() {
+      return {
+        networks: [
```

Comment:
This isn't able to get the mapped getter and doesn't work in the data section, if 'networks' is a computed property it'll work properly.
There are also some unit tests failing because of this.

Reply:
thanks I have moved it to a computed property and unit tests are passing